index.html

```diff
@@ -9298,6 +9298,279 @@
         </section>
       </div>
     </section>
+    <section id="rtcquicstream-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+interface RTCQuicStream {
+    readonly        attribute RTCQuicTransport    transport;
+    readonly        attribute RTCQuicStreamState  state;
+    readonly        attribute unsigned long       bufferedAmount;
```

Comment:
Let's keep approaches to buffering in a separate PR.

Reply:
Removed bufferedAmount.